### PR TITLE
etcd/tls-etcd-clients: Polish new document

### PR DESCRIPTION
**Pull request by joshix**
Atop #674.

Rename for concision and sanity.
Make one way - cloud-config - to do each step, instead of two ways.
Rewrite for style/language.

Before merge: New file needs .top frontmatter file in coreos-inc/coreos-pages.

### CODE REVIEW

**Comment by joshix** [comment]
@kayrus @robszumski The original PR had two ways to do each step; this changeset documents only ONE way, cloud-config, to do each. Please review that in particular. It is an attempt to simplify the instruction stream. I appreciate that each of the files discussed can also be created manually, but I don't see the advantage of documenting the 2 ways to do each step in this how-to.

**Comment by joshix** [comment]
also @omkensey from previous PR discussion

**Comment by kayrus** [comment]
LGTM

File: etcd/tls-etcd-clients.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```yaml
+coreos:
+  flannel:
+    etcd_endpoints: "https://172.16.0.101:2379,https://172.16.0.102:2379,https://172.16.0.103:2379"
+    etcd_cafile: /etc/ssl/etcd/ca.pem
+    etcd_certfile: /etc/ssl/etcd/client.pem
+    etcd_keyfile: /etc/ssl/etcd/client-key.pem
+```
+
+will generate the following `/run/flannel/options.env`:
```
Comment (robszumski) [inline etcd/tls-etcd-clients.md]: will create a [systemd drop-in](https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html)

Comment (robszumski) [inline etcd/tls-etcd-clients.md]: whoops, this isn't a dropin

File: etcd/tls-etcd-clients.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+```yaml
+coreos:
+  fleet:
+    etcd_servers: "https://172.16.0.101:2379,https://172.16.0.102:2379,https://172.16.0.103:2379"
+    etcd_cafile: /etc/ssl/etcd/ca.pem
+    etcd_certfile: /etc/ssl/etcd/client.pem
+    etcd_keyfile: /etc/ssl/etcd/client-key.pem
+```
+
+will generate a `/run/systemd/system/fleet.service.d/20-cloudinit.conf` drop-in with these contents:
```
Comment (robszumski) [inline etcd/tls-etcd-clients.md]: link to drop in: https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html

File: etcd/tls-etcd-clients.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+```yaml
+coreos:
+  locksmith:
+    endpoint: "https://172.16.0.101:2379,https://172.16.0.102:2379,https://172.16.0.103:2379"
+    etcd_cafile: /etc/ssl/etcd/ca.pem
+    etcd_certfile: /etc/ssl/etcd/client.pem
+    etcd_keyfile: /etc/ssl/etcd/client-key.pem
+```
+
+This results in a
```
Comment (robszumski) [inline etcd/tls-etcd-clients.md]: Mention drop in here too

**Comment by joshix** [comment]
@robszumski Added a link and explicit identification of the drop-ins. The first one is an environment file instead; stet.

**Pull request merged**